### ISSUE TITLE
Add `Convert to 'package:' import for everywhere in file` context action

### ISSUE THREAD

**Issue by Luckey-Elijah** [Dart-Code/Dart-Code#3823]
**Is your feature request related to a problem? Please describe.**
When Working on fixing up old code that uses relative imports, many times it uses relative imports in the dart source code.

**Describe the solution you'd like**
I would like a context action that will `Convert to 'package:' import for everywhere in file`. Similar to what screen-capped is below.
<img width="459" alt="Screen Shot 2022-02-08 at 10 07 32 AM" src="https://user-images.githubusercontent.com/38171761/153014971-d502174b-afa4-4562-8495-b74ab61fe147.png">

**Describe alternatives you've considered**
Using the same `Convert to 'package:' import` actions several times on a file import directives.

**Comment by DanTup** [comment]
I've got a fix in progress here:

https://dart-review.googlesource.com/c/sdk/+/232265/

<img width="478" alt="Screenshot 2022-02-10 at 09 56 00" src="https://user-images.githubusercontent.com/1078012/153382839-957d60e3-355a-4e5a-9181-4d4dc90c5d4f.png">

The "Convert to 'package:' import" shown in your original screenshot is an _assist_ and not a _fix_ (which is why there's no bulk option, and it's not at the top of the list). The change applies the existing fix (of the same name) to the lint `always_use_package_imports` which both moves them to the top and enables bulk fixing.

**Comment by DanTup** [comment]
Note: Since the fix is in the SDK, it'll actually show up in a future Dart/Flutter SDK release and not a VS Code extension release (these fixes are powered by the Dart language server in the SDK).

I'll close this issue once that change is merged into the SDK.
